### PR TITLE
[SPVN-25 ]AcceptanceTest scenarios for player registration

### PR DESCRIPTION
**Pull request by vasumathy**
[SPVN-25] - Fixing review comments
- Updated error status with exact status code
-  Ignored Clubcard validation since its does not happens in SPINE

- Acceptance test scenarios for PlayerRegistration endpoint are created
- Error messages required update[Will be updated after the release of new service]

### CODE REVIEW

File: src/Bede.SPINE.AcceptanceTest/Features/PlayerRegistration.feature [hunk 1]
```diff
@@ -0,0 +1,105 @@
+﻿Feature: PlayerRegistration
+
+@ignore
+Scenario Outline: Successful Player Registration Using Valid Player Details In Multiple SiteCode
+	Given a valid token is generated for a player in sitecode < Sitecode_Value >
+	And valid player details are set for registration
+	When player registration request is sent to RestSharp
+	Then the response status is OK
+	And the repsone returns a unique player id
```
Comment (joseph-norman) [inline src/Bede.SPINE.AcceptanceTest/Features/PlayerRegistration.feature]: typo here

File: src/Bede.SPINE.AcceptanceTest/Features/PlayerRegistration.feature [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	| Field_Name   |
+	| FirstName    |
+	| LastName     |
+	| Password     |
+	| DateOfBirth  |
+	| CampaignData |
+	| AddressLine1 |
+	| CountryCode  |
+	| IpAddress    |
+	| ClubCardNo   |
```
Comment (pmcatominey) [inline src/Bede.SPINE.AcceptanceTest/Features/PlayerRegistration.feature]: I believe the ClubCardNo is within SiteData and so it won't be subject to validation by spine, it's designed to hold arbitrary site specific profile values.

**Comment by joseph-norman** [comment]
General question - in the invalid responses (i.e `bad request`), do we want to verify the status codes? In fact, do we want to verify the status codes for all scenarios? Some of them return endpoint-specific codes such as registration (which when successful returns a 201).

**Comment by vasumathy** [comment]
:rabbit:

**Comment by vasumathy** [comment]
:rabbit:

**Pull request merged**